pages/common/flutter.md

```diff
@@ -31,6 +19,14 @@
 
 `flutter run -d all`
 
+- Download all packages specified in `pubspec.yaml`:
```

Comment:
```suggestion
- Download all the packages listed by the `pubspec.yaml` file in the local path:
```

Reply:
They're not "listed by it" - `pubspec` is your configuration file, that *you* specify. This example should be left unchanged :confused:

Reply:
I agree. Let's leave this example as-is.